src/main/java/org/prebid/server/analytics/reporter/greenbids/model/CommonMessage.java

```diff
@@ -0,0 +1,39 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import org.prebid.server.auction.model.AuctionContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommonMessage {
+
+    public String version;
+    public String auctionId;
+    public String referrer;
+    public double sampling;
+    public String prebid;
+    public String greenbidsId;
+    public String pbuid;
+    public String billingId;
+    public List<AdUnit> adUnits;
+    public Long auctionElapsed;
+
+    public CommonMessage(
+            AuctionContext auctionContext,
+            GreenbidsConfig greenbidsConfig,
+            Long auctionElapsed,
+            String greenbidsId,
+            String billingId
+    ) {
```

Comment:
self-review: I did not added `ortb2Imp` field as on the side of Greenbids Analytics server it is marked as deprecated non standard field used to support custom GB analytics payload. Need think about whether to add it to the paylod